neuron_reduce/reducing_methods.py

```diff
@@ -30,259 +39,327 @@
     return lowest_imp_vec
 }''')
 
-RC = None
-RM = None
-RA = None
-q = None
 
-has_apical = None
-frequency = None
+@contextlib.contextmanager
```

Comment:
helper so h.pop_section() isn't overlooked

Reply:
Just need to make sure that we really want to pop the section each time...

Reply:
yup, I think there is a case where I don't use this, b/c of the different push/pop